scripts/circle-ci/publish-images-to-antithesis.sh

```diff
@@ -15,31 +10,22 @@ if [[ -z "${ANTITHESIS_LOGIN_JSON}" ]]; then
   exit 1
 fi
 
-if [[ -z "${ANTITHESIS_WEBHOOK_PASSWORD}" ]]; then
-  echo "Antithesis webhook password is not set as an environment variable, exiting."
-  exit 1
-fi
-
 printenv ANTITHESIS_LOGIN_JSON | base64 -d | docker login -u _json_key https://${ANTITHESIS_REPO_URL} --password-stdin
 
 ./gradlew --scan dockerTag
 
 VERSION=$(./gradlew -q printVersion)
+EXPECTED_ANTITHESIS_TAG="latest"
 
 docker pull palantirtechnologies/cassandra:2.2.18-1.116.0
-docker tag palantirtechnologies/cassandra:2.2.18-1.116.0 ${ANTITHESIS_REPO_URL}/cassandra:2.2.18-1.116.0
-docker push ${ANTITHESIS_REPO_URL}/cassandra:2.2.18-1.116.0
-
-docker tag palantirtechnologies/timelock-server-distribution:${VERSION} ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
-docker push ${ANTITHESIS_REPO_URL}/timelock-server-distribution:${1}
+docker tag palantirtechnologies/cassandra:2.2.18-1.116.0 ${ANTITHESIS_REPO_URL}/cassandra:${EXPECTED_ANTITHESIS_TAG}
+docker push ${ANTITHESIS_REPO_URL}/cassandra:${EXPECTED_ANTITHESIS_TAG}
```

Comment:
for cassandra specifically, should we be on unspecified or latest? This implies latest, but want to double check

Reply:
We should publish with tag `latest` but leave `unspecified` in the actual docker-compose.yml, according to antithesis folks.